src/test/java/org/jboss/logmanager/handlers/SocketHandlerTests.java

```diff
@@ -115,71 +111,81 @@ public void testTcpPortChange() throws Exception {
     @Test
     public void testProtocolChange() throws Exception {
         Assume.assumeFalse(JDK_8219991_ERROR_MESSAGE, JDK_8219991);
-        try (SocketHandler handler = createHandler(Protocol.TCP)) {
```

Comment:
This `try` should not go way. There should try-with-resources can be removed, but we need the `handler.close()` to be in a `finally` block in case there is a failure.

Reply:
try and finally block added.